client/src/apps/code-collabo/styles/app.main.js

```diff
@@ -68,6 +86,19 @@ export default css.global`
             border-right: 1px solid var(--color-neutral-200);
         }
 
+        .app__logo {
+            margin-bottom: 82px;
+        }
+
+        .app__menubar__nav__link {
+            padding: 15px 0px;
```

Comment:
Desktop Padding top and bottom - 16, not 15.

Reply:
@Ifycode Done